kafka-links.csv

```diff
@@ -116,6 +117,7 @@ Conduktor;https://www.conduktor.io/
 Strimzi Kafka Bridge;https://strimzi.io/docs/bridge/latest/
 Confluent REST Proxy;https://docs.confluent.io/platform/current/kafka-rest/index.html
 Kafka-Pixy;https://github.com/mailgun/kafka-pixy
+Kong Kafka Upstream;https://docs.konghq.com/hub/kong-inc/kafka-upstream/
```

Comment:
The book addition above is great; congrats on publishing it!

```suggestion
```

This one appears to be rather specific, I feel it may be a bit too focused in terms of scope for this map, so I'm going to exclude this for now.